kata-design-requirements.md

```diff
@@ -0,0 +1,82 @@
+## Mandatory requirements
+
+The Kata Containers runtime **MUST** fulfill all requirements below:
+
+### OCI compatibility
+The Kata Containers runtime **MUST** implement the OCI runtime specification and support all the OCI runtime operations.
+
+### `runc` CLI compatibility
+In theory, being OCI compatible should be enough. In practice the a Kata Containers runtime should comply the latest *stable* `runc` CLI. In particular, it **MUST** implement with the following `runc` commands:
+
+* `create`
+* `delete`
+* `exec`
+* `kill`
+* `list`
+* `pause`
+* `ps`
+* `state`
+* `start`
+* `version`
+
+and the following command line options:
+* `--console-socket`
+* `--pid-file`
+
+### CRI and Kubernetes support
+The Kata Containers project **MUST** provide 2 interfaces for CRI shims to be able to manage hardware virtualization based Kubernetes pods and containers:
```

Comment:
I'd also write "2" as "two", or just drop it to avoid having to maintain that bit and just say:

> **MUST** provide the following interfaces for ...

Reply:
agreed.